bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -512,6 +512,18 @@ pub trait RoomInfoListener: Sync + Send {
     fn call(&self, room_info: RoomInfo);
 }
 
+#[derive(uniffi::Record)]
+pub struct ExportedRoomMember {
+    user_id: String,
+    display_name: Option<String>,
+    avatar_url: Option<String>,
+    membership: MembershipState,
+    is_name_ambiguous: bool,
+    power_level: i64,
+    normalized_power_level: i64,
+    is_ignored: bool,
+}
+
```

Comment:
Can you put this into the room_member.rs file instead, along the `RoomMember` struct?